server/datastore/mysql/statistics.go

```diff
@@ -84,6 +84,12 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		stats.VulnDetectionEnabled = appConfig.VulnerabilitySettings.DatabasesPath != ""
 		stats.SystemUsersEnabled = appConfig.Features.EnableHostUsers
 		stats.HostsStatusWebHookEnabled = appConfig.WebhookSettings.HostStatusWebhook.Enable
+		stats.MDMEnabledAndConfigured = appConfig.MDM.EnabledAndConfigured
+		stats.HostExpiryEnabled = appConfig.HostExpirySettings.HostExpiryEnabled
+		stats.JiraIntegrationEnabled = len(appConfig.Integrations.Jira) > 0
```

Comment:
```suggestion
		stats.JiraIntegrationEnabled = len(appConfig.Integrations.Jira) > 0
```

Can we collect the number of integration just as easily? If yes, let's do that and calls this `NumJiraIntegrations`

Why? `JiraIntegrationEnabled` isn't necessarily true if they just add an integration. They have to go enable it on **Software** or **Policies** page.

Reply:
More generally, I think we should find a better way to track "is the user using Jira integration"

To move quickly, I think we cut this one entirely for now and come back to this one.

Reply:
Makes sense.
I'll comment it out for now and add a TODO comment for the future.